### DISCUSSION TITLE
Problem installing the inventree-brother.plugin

### DISCUSSION THREAD

**Discussion by SergeoLacruz** [inventree/InvenTree#3131] [Q&A] [answered]
Hi,
I am running 0.8.0 dev. I installed the inventree-brother-plugin  simply using pip install inventree-brother-plugin. It is in .local/lib/python3.9/site-packages. When I start the server using invoke server -a x.x.x.x I see a message telling
"deprecation warning: brother_ql.devicedependent is deprecated and will be removed in a future release" which is probably not a problem so far.
But in the inventree plugin admin page I see just an empty line as shown here:

https://github.com/SergeoLacruz/inventree-zebra-plugin/blob/main/plugin.png

What am I doing wrong?

Michael

**Comment by SchrodingersGat** [comment, marked as answer]
Installing via pypi won't work, due to the fact that:

- inventree-brother-plugin relies on a specific commit of the brother_ql library
- pypi does not let you have non-pypi dependencies

So you will have to install using the methods described in the readme for [inventree-brother-plugin](https://github.com/inventree/inventree-brother-plugin)

It looks like you are writing a zebra plugin?

**Comment by SergeoLacruz** [comment]
Hi,
That works :-) thanks for the hint. 
But writing a new one seems to be more complicated than expected... I tried this

https://github.com/SergeoLacruz/inventree-zebra-plugin

But gut the same problem as above (empty line). It seems I am missing something. 

Michael

**Reply by SchrodingersGat** [reply]
Remove the PLUGIN_ prefix from

- `PLUGIN_NAME`
- `PLUGIN_SLUG`
- `PLUGIN_TITLE`

**Comment by SergeoLacruz** [comment]
Morning,
This works. Thanks for the help. I can install it now and print :-)
But honestly, I have no clue why...
In the brother plugin the prefixes are there and it also works. Some more info is needed for a deeper understanding.
I will work on the code now.

Michael
